### ISSUE TITLE
Doc examples: assertion error on nnzj?

### ISSUE THREAD

**Issue by jbcaillau** [exanauts/ExaModels.jl#104]
@frapac @sshin23 Naïve question here: not been able to run either [quadrotor](https://exanauts.github.io/ExaModels.jl/dev/quad/) or [column](https://exanauts.github.io/ExaModels.jl/dev/distillation/) examples as a I run into the error below (same error in both case):
```julia
julia> m = quadrotor_model(100)
ERROR: AssertionError: nnzj == lin_nnzj + nln_nnzj
Stacktrace:
 [1] NLPModels.NLPModelMeta{…}(nvar::Int64; x0::Vector{…}, lvar::Vector{…}, uvar::Vector{…}, nlvb::Int64, nlvo::Int64, nlvc::Int64, ncon::Int64, y0::Vector{…}, lcon::Vector{…}, ucon::Vector{…}, nnzo::Int64, nnzj::Int64, lin_nnzj::Int64, nln_nnzj::Int64, nnzh::Int64, lin::Vector{…}, minimize::Bool, islp::Bool, name::String)
   @ NLPModels ~/.julia/packages/NLPModels/e5o8i/src/nlp/meta.jl:148
 [2] NLPModelMeta
   @ ~/.julia/packages/NLPModels/e5o8i/src/nlp/meta.jl:119 [inlined]
 [3] NLPModelMeta
   @ ~/.julia/packages/NLPModels/e5o8i/src/nlp/meta.jl:219 [inlined]
 [4] ExaModel(c::ExaCore{Float64, Vector{Float64}, Nothing}; prod::Nothing)
   @ ExaModels ~/.julia/packages/ExaModels/tBDe7/src/nlp.jl:228
 [5] ExaModel
   @ ~/.julia/packages/ExaModels/tBDe7/src/nlp.jl:227 [inlined]
 [6] quadrotor_model(N::Int64; backend::Nothing)
   @ Main ./REPL[2]:79
 [7] quadrotor_model(N::Int64)
   @ Main ./REPL[2]:1
 [8] top-level scope
   @ REPL[4]:1
Some type information was truncated. Use `show(err)` to see complete types.
```

Note: the env used for testing only contains `ExaModels` and `NLPModelsIpopt`:
```julia
(@exa) pkg> st
Status `~/.julia/environments/exa/Project.toml`
  [1037b233] ExaModels v0.7.1
  [f4238b75] NLPModelsIpopt v0.10.2
```

**Comment by jbcaillau** [comment]
NB. @frapac @sshin23 Actually, I am getting the same error even with the [Getting started](https://exanauts.github.io/ExaModels.jl/dev/guide)

**Comment by sshin23** [comment]
Thanks for reporting this. We're currently working on the fix in #105

**Comment by jbcaillau** [comment]
> Thanks for reporting this. We're currently working on the fix in #105

@sshin23 FWIW: I am downgrading to NLPModels v0.21.1 to remove the `@assert` on `nnzj` and run the tests

**Comment by sshin23** [comment]
This is now fixed with the NLPModels v0.21.3 release
